gitpod-io/gitpod-vscode-desktop
# Send proper telemetry when copy needed files

## Description

> We enabled local ssh config for all users, we should do it only if local ssh experiment is enabled.
> Tracking of analytics event don't match to number of failed exceptions, let's fire event and exception at the same point, i.e. initialized.

- Enable LocalSSHService only when we have feature flag enabled
- Fix update tmp folder partly [[1](https://github.com/gitpod-io/gitpod-vscode-desktop/pull/69/files#diff-d9ccc73eceac9997c215de8b66762f732c45d56bd5ed1fffcd7a70ef66abf176R691-R692)]

## Related Issue(s)

Related [IDE-97](https://linear.app/gitpod/issue/IDE-97)
Fixed [IDE-128](https://linear.app/gitpod/issue/IDE-128), [IDE-100](https://linear.app/gitpod/issue/IDE-100), [IDE-132](https://linear.app/gitpod/issue/IDE-132)

## How to test

- Run extension from source
- It should failed if you change `remote.ssh.configFile` to a unaccessable location

Test result
<img width="433" alt="image" src="https://github.com/gitpod-io/gitpod-vscode-desktop/assets/20944364/9341f4fc-8383-4977-a152-71825bb8b2fc">

![image](https://github.com/gitpod-io/gitpod-vscode-desktop/assets/20944364/c29f5474-1c63-4504-9a35-53cf79d9162a)
